### ISSUE TITLE
Default example does not work

### ISSUE THREAD

**Issue by adrian2x** [whitead/paper-qa#102]
Tried to set up using the default example from the README.

Got this error:
```
WARNING! model is not default parameter.
                    model was transferred to model_kwargs.
                    Please confirm that model is what you intended.
Traceback (most recent call last):
  File "/Users/adrian/src/test/paper_qa.py", line 4, in <module>
    docs = Docs()
           ^^^^^^
  File "/Users/adrian/src/test/venv/lib/python3.11/site-packages/paperqa/docs.py", line 60, in __init__
    self.update_llm(llm, summary_llm)
  File "/Users/adrian/src/test/venv/lib/python3.11/site-packages/paperqa/docs.py", line 78, in update_llm
    llm = ChatOpenAI(temperature=0.1, model=llm)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ChatOpenAI
__root__
  Parameters {'model'} should be specified explicitly. Instead they were passed in as part of `model_kwargs` parameter. (type=value_error)
```

**Comment by nicolascoffman** [comment]
Getting the same issue.  Found this example:
` from paperqa import Docs

docs = paperqa.Docs()`
Results in:
`Traceback (most recent call last):
  File "C:\Users\.....\annotaded-bib.py", line 12, in <module>
    docs = paperqa.Docs()
NameError: name 'paperqa' is not defined`

But I think it should be:

` from paperqa import Docs

docs = Docs()`

Which results in error from @adrian2x.

**Comment by ghicvirgil** [comment]
Use model_name instead of model -> ChatOpenAI(model_name="gpt-3.5-turbo")

**Comment by OliverOffing** [comment]
There was [an upstream breaking change](https://github.com/hwchase17/langchain/pull/4366/files#r1192660713). The quick fix is to add `langchain==0.0.162` to your `requirements.txt`.

@whitead You should consider hard coding a specific version of `langchain`, given that they don't seem to be following semver.

```diff
-  "langchain>=0.0.157", 
+  "langchain==0.0.157", 
```

https://github.com/whitead/paper-qa/blob/89c8ea8137127f18a0e12189350a72b9abe503ff/setup.py#L19
